pkgs/by-name/as/aseprite/package.nix

```diff
@@ -116,15 +119,16 @@ clangStdenv.mkDerivation (finalAttrs: {
     "-DUSE_SHARED_LIBPNG=ON"
     "-DUSE_SHARED_LIBWEBP=ON"
     "-DUSE_SHARED_PIXMAN=ON"
-    "-DUSE_SHARED_TINYXML=ON"
+    #"-DUSE_SHARED_TINYXML=ON"
```

Comment:
Have you forgotten to remove this completely?

Reply:
Not sure for now.

Reply:
Then, maybe mark this PR as a draft then.